README.md

```diff
@@ -30,28 +29,17 @@
 Introduction
 ------------
 
-**[LevelDB](https://github.com/google/leveldb)** is a simple key-value store built by Google, inspired by BigTable. It's used in Google Chrome and many other products. LevelDB supports arbitrary byte arrays as both keys and values, singular *get*, *put* and *delete* operations, *batched put and delete*, bi-directional iterators and simple compression using the very fast [Snappy](http://google.github.io/snappy/) algorithm.
+**`levelup` is a friendly Node.js interface to key-value stores that follow the characteristics of [LevelDB](https://github.com/google/leveldb). It is the core of the [Level ecosystem](https://github.com/level/awesome).**
```

Comment:
Opening sentence is too long

Reply:
I think we use bold way too much in general.

Reply:
Agreed. Perhaps only the first paragraph and the last (about `level`) need to be bold?